jetstream/analysis.py

```diff
@@ -263,10 +268,9 @@ def check_runnable(self, current_date: Optional[datetime] = None) -> bool:
         if self.config.experiment.start_date is None:
             raise errors.NoStartDateException(self.config.experiment.normandy_slug)
 
-        if (
-            current_date
-            and self.config.experiment.end_date
-            and self.config.experiment.end_date < current_date
+        if current_date and (
+            self.config.experiment.end_date is None
```

Comment:
an experiment with a null endDate has not ended, so this doesn't seem right to me

(it's starting to feel bad that we have to reason about this in so many places)